src/Adapter/Category/Repository/CategoryRepository.php

```diff
@@ -184,13 +242,14 @@ public function getBreadcrumb(CategoryId $categoryId, LanguageId $languageId, st
             ->setParameter('languageId', $languageId->getValue())
         ;
 
-        $result = $qb->execute()->fetchAll();
+        $results = $qb->execute()->fetchAllAssociative();
+
         $parentNames = [];
-        foreach ($result as $category) {
+        foreach ($results as $category) {
```

Comment:
array_column ?

Reply:
ah nice, didn't actually know about this method :smile: :+1: